game/main.py

```diff
@@ -22,13 +22,26 @@ def move(self, keys):
         global LAST_POSITION
         LAST_POSITION = self.pos_x, self.pos_y
         if keys[pygame.K_LEFT]:
-            self.pos_x -= self.speed
+            if self.pos_x > self.speed:
+                self.pos_x -= self.speed
+            else:
+                self.pos_x = self.pos_x
         if keys[pygame.K_RIGHT]:
-            self.pos_x += self.speed
+            if self.pos_x <= screen_width - 60:
```

Comment:
Por que esse valor de 60?

Reply:
Esse valor corresponde a borda que idealizei como "padrão" visual, existe um espaçamento de 5px maior que o size...

Não utilizei o self.size para operar o valor por que também teria que utilizar um número para manter a borda de 5px.

Reply:
pergunta numero 1:  Pq tem que ter borda?
pergunta numero 2: E se mudar o. tamanho do retangulo? tipo se for ter um chefao que ele é maior?

logo nao é pra ter borda e é pra isso funcionar independente do tamanho do player

Reply:
Entendi, não precisa de borda, e o número limita a usabilidade do código... realizado correção.